packages/mermaid/src/diagrams/er/erRenderer.js

```diff
@@ -643,9 +643,15 @@ export const draw = function (text, id, _version, diagObj) {
   svg.attr('viewBox', `${svgBounds.x - padding} ${svgBounds.y - padding} ${width} ${height}`);
 }; // draw
 
+/** UUID namespace for ER diagram IDs */
```

Comment:
Instead of computing this, we could actually go to https://www.uuidgenerator.net/ and generate a unique UUID and use the actual UUID here.

The UUID could be a v5 based on something like timestamp +
      'https://mermaid-js.github.io/mermaid/syntax/entityRelationshipDiagram.html' +
     mermaid version

Would save a wee bit of client-side time.

Reply:
`28e9f9db-3c8d-5aa5-9faf-44286ae5937c` is the output of
```ts
uuid5(
  'https://mermaid-js.github.io/mermaid/syntax/entityRelationshipDiagram.html',
  uuid5.URL
);
```

We could hard-code that, with the generation code as a comment above. 

I don't think we should be using timestamp or mermaid version in the UUID generation, as the requirement is for it to generate the same UUID for the same input string.

Reply:
Doh - you're totally correct, @sidharthv96  :-)

Reply:
Fixed in https://github.com/mermaid-js/mermaid/pull/3916/commits/7855edae6babc14f9de9dd6de070d71e6800ba53

It's always great when I can label a commit as `perf:` for performance!

I added both of you as Co-authors to the commit!

> I don't think we should be using timestamp or mermaid version in the UUID generation, as the requirement is for it to generate the same UUID for the same input string.

Agreed, the UUID namespace doesn't really matter, to be honest, as long as it's consistent, and is unique within each ER diagram.